node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoreCollector.java

```diff
@@ -49,6 +52,17 @@ Path readBinPathFallback(NodeAgentContext context, Path coredumpPath) {
         return Paths.get(matcher.group("path").split(" ")[0]);
     }
 
+    Path GDBPath(NodeAgentContext context) {
+        DockerImage image = context.node().currentDockerImage().get();
+
+        if (image.tag().get().startsWith("vespa/ci")) {
+            return vespaHomeInContainer.resolve("bin64/gdb");
```

Comment:
I suggest using `context.pathInNodeUnderVespaHome()` and remove the option from constructor